### PR TITLE
Print include statement when modules install

### LINKED ISSUE #1420: modules install to also print/write `include` statements
### Description of feature

print `include` statements to terminal for easy copy to destination files

### PR DESCRIPTION
**Pull request by asthara10**
Closes #1420

When using `nf-core modules install` print the include statements to terminal for easy copy to destination files.

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: nf_core/modules/install.py [hunk 1]
```diff
@@ -130,6 +130,14 @@ def install(self, module):
         if not self.download_module_file(module, version, self.modules_repo, install_folder):
             return False
 
+        # Print include statement
+        if module == "stringtie/stringtie":
+            # Only with stringtie the process name is STRINGTIE instead of STRINGTIE_STRINGTIE
```
Comment (drpatelh) [inline nf_core/modules/install.py:135]: This is a bug. We should update the nf-core module to use `STRINGTIE_STRINGTIE`.

Comment (asthara10) [reply nf_core/modules/install.py:135]: I see. I can modify the module, but we should update all the pipelines that use that module then, right?

Comment (drpatelh) [reply nf_core/modules/install.py:135]: Modifying just the module is great. When developers re-install the module they can update the pipeline code themselves.

Comment (drpatelh) [reply nf_core/modules/install.py]: Update the module here https://github.com/nf-core/modules/pull/1546 @asthara10 if you fancy reviewing?

**Review by ggabernet — APPROVED** [review]
Looks good to me now!

**Comment by drpatelh** [comment]
Yes, sorry, I wanted to test this branch to see how the `log.info` for the `include` was printed to screen but that's a minor change if we need to make it.

**Pull request merged**